### PR TITLE
Add user tagging in running notes

### PR DESCRIPTION
**Pull request by aanil**
Added user tagging with '@user.name' in running notes. Users will be shown in a dropdown and tagged users will be mailed with note contents when it is submitted. 

Uses en external javascript library called 'jquery-sew' which I've forked (personal repo) and modified a bit to remove dependencies and fit our requirements.

### CODE REVIEW

File: run_dir/design/running_notes_tab.html [hunk 1]
```diff
@@ -61,3 +61,5 @@ <h4>Preview</h4>
         <!-- display running notes -->
         <div class="panel-group" id="running_notes_panels"></div>
     </div>
+    <script src="/static/js/jquery.caretposition.js"></script>
+		<script src="/static/js/jquery.sew.js"></script>
```
Comment (alneberg) [inline run_dir/design/running_notes_tab.html]: annoying whitespace... 😅

Comment (aanil) [reply run_dir/design/running_notes_tab.html]: Should be fixed now!

File: run_dir/static/css/status.css [hunk 2]
```diff
@@ -1144,3 +1144,38 @@ input.quantity_updateable {
...
+}
+
+.-sew-list {
+	list-style: none;
+	margin: 0;
+	padding: 2px;
+	max-height: 100px;
+  min-width: 120px;
+	overflow-x: hidden;
+  overflow-y: auto;
```
[the remark this answers is not in the document]
Comment (aanil) [reply run_dir/static/css/status.css]: Should be fixed now!

File: run_dir/static/css/status.css [hunk 3]
```diff
@@ -1144,3 +1144,38 @@ input.quantity_updateable {
...
+	box-shadow: 0 0 1px rgba(0, 0, 0, 0.1);
+	min-width: 200px;
+}
+
+.-sew-list {
+	list-style: none;
+	margin: 0;
+	padding: 2px;
+	max-height: 100px;
+  min-width: 120px;
```
[the remark this answers is not in the document]
Comment (aanil) [reply run_dir/static/css/status.css]: Should be fixed now!

File: run_dir/static/js/running_notes.js [hunk 4]
```diff
@@ -47,11 +56,11 @@ function make_running_note(date, note){
     noteText = '<pre>'+make_project_links(note['note'])+'</pre>';
     var datestring = '?';
   }
-  
+  var printHyphen =category? ' - ': ' ';
```
[the remark this answers is not in the document]
Comment (aanil) [reply run_dir/static/js/running_notes.js]: 😄

File: run_dir/static/css/status.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (alneberg) [reply run_dir/static/css/status.css]: B E A Utiful

**Pull request merged**